### PR TITLE
[feat][AttributeModal]: add attribute description to field modal in content builder

### PR DESCRIPTION
**Pull request by joshuaellis**
### What does it do?

Adds the attributes description to the SubHeader of the modal

### How to test it?

* Go to Content Builder
* Add a field
* Select an attribute
* View the modal and see the attribute description

### Related issue(s)/PR(s)

* resolves CONTENT-239

### CODE REVIEW

**Review by jhoward1994 — DISMISSED** [review]
Nice, looks good!

File: packages/core/content-type-builder/admin/src/components/FormModalSubHeader/index.js [hunk 1]
```diff
@@ -1,11 +1,23 @@
...
 import upperFirst from 'lodash/upperFirst';
+import { Flex } from '@strapi/design-system/Flex';
+import { Typography } from '@strapi/design-system/Typography';
+
 import { getTrad } from '../../utils';
+
 import getModalTitleSubHeader from './getModalTitleSubHeader';
 
+const SubHeaderContainer = styled(Flex)`
+  margin-bottom: 10px;
```
Comment (soupette) [inline packages/core/content-type-builder/admin/src/components/FormModalSubHeader/index.js:14]: Please use token from the theme instead, `10px` is not a valid token

File: packages/core/content-type-builder/admin/src/components/FormModalSubHeader/index.js [hunk 2]
```diff
@@ -1,11 +1,23 @@
...
+`;
+
+const Subtitle = styled(Typography)`
```
[the remark this answers is not in the document]
Comment (joshuaellis) [reply packages/core/content-type-builder/admin/src/components/FormModalSubHeader/index.js:17]: This adds unnecessary dom, I think it's better to use a ThemeToken.

**Review by jhoward1994 — DISMISSED** [review]

File: packages/core/content-type-builder/admin/src/components/FormModalSubHeader/index.js [hunk 3]
```diff
@@ -1,11 +1,23 @@
...
 import upperFirst from 'lodash/upperFirst';
+import { Flex } from '@strapi/design-system/Flex';
+import { Typography } from '@strapi/design-system/Typography';
+
 import { getTrad } from '../../utils';
+
 import getModalTitleSubHeader from './getModalTitleSubHeader';
 
+const SubHeaderContainer = styled(Flex)`
+  margin-bottom: ${(props) => props.theme.spaces[2]};
```
Comment (ronronscelestes) [inline packages/core/content-type-builder/admin/src/components/FormModalSubHeader/index.js:14]: mmh I think we could use `paddingBottom` prop, and to handle the space between `Subtitle` and the other `Typography` we could base `SubHeaderContainer` on `Stack` component 👀

```js
<Stack direction="column" alignItems="flex-start" paddingBottom={2} spacing={1}>
  <Typography variant="beta" />
  <Typography variant="pi" />
</Stack>
```

**Review by ronronscelestes — COMMENTED** [review]
Neat 💃 
Small comment about DS

**Pull request merged**